### PR TITLE
OCPBUGS#30018 change to the 4.12 EOL phase date in 4.14 RNs

### PR DESCRIPTION
**Pull request by cbippley**
Version(s): 4.14

Issue: [OCPBUGS-30018](https://issues.redhat.com/browse/OCPBUGS-30018)

Link to docs preview: [Preview](https://72335--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-14-release-notes#ocp-4-14-about-this-release)

QE review: 
- [X] QE has approved this change.

Additional information: Kathryn okayed this to go ahead without change management.

### CODE REVIEW

File: release_notes/ocp-4-14-release-notes.adoc [hunk 1]
```diff
@@ -33,7 +33,7 @@ Starting with {product-title} {product-version}, Extended Update Support (EUS) i
 {product-title} {product-version} is an Extended Update Support (EUS) release. More information on Red Hat OpenShift EUS is available in link:https://access.redhat.com/support/policy/updates/openshift#ocp4_phases[OpenShift Life Cycle] and link:https://access.redhat.com/support/policy/updates/openshift-eus[OpenShift EUS Overview].
 
 //TODO: The line below should be used when it is next appropriate. Revisit in August 2023 time frame.
-Maintenance support ends for version 4.12 on 25 January 2025 and goes to extended life phase. For more information, see the link:https://access.redhat.com/support/policy/updates/openshift[Red Hat {product-title} Life Cycle Policy].
+Maintenance support ends for version 4.12 on 17 January 2025 and goes to extended life phase. For more information, see the link:https://access.redhat.com/support/policy/updates/openshift[Red Hat {product-title} Life Cycle Policy].
```
Comment (liangxia) [inline release_notes/ocp-4-14-release-notes.adoc:36]: From https://access.redhat.com/support/policy/updates/openshift,
Maintenance support ends for version 4.12 on July 17, 2024, Extended update support ends on January 17, 2025

Comment (cbippley) [reply release_notes/ocp-4-14-release-notes.adoc]: @liangxia How does it look now?

File: release_notes/ocp-4-14-release-notes.adoc [hunk 2]
```diff
@@ -32,8 +32,8 @@ Starting with {product-title} {product-version}, Extended Update Support (EUS) i
 //TODO: Add the line below for EUS releases.
 {product-title} {product-version} is an Extended Update Support (EUS) release. More information on Red Hat OpenShift EUS is available in link:https://access.redhat.com/support/policy/updates/openshift#ocp4_phases[OpenShift Life Cycle] and link:https://access.redhat.com/support/policy/updates/openshift-eus[OpenShift EUS Overview].
 
-//TODO: The line below should be used when it is next appropriate. Revisit in August 2023 time frame.
-Maintenance support ends for version 4.12 on 25 January 2025 and goes to extended life phase. For more information, see the link:https://access.redhat.com/support/policy/updates/openshift[Red Hat {product-title} Life Cycle Policy].
+//TODO: The line below should be used when it is next appropriate.
+Maintenance support ends for version 4.12 on 17 July 2025 and goes to extended update support phase. For more information, see the link:https://access.redhat.com/support/policy/updates/openshift[Red Hat {product-title} Life Cycle Policy].
```
Comment (liangxia) [inline release_notes/ocp-4-14-release-notes.adoc:36]: From https://access.redhat.com/support/policy/updates/openshift,
Maintenance support ends for version 4.12 on **17 July 2024**

**Pull request merged**